configuration/spotbugs/spotbugs-exclude.xml

```diff
@@ -90,6 +90,95 @@
+    <Match>
+		<Class name="~org\.openjdk\.jmc\..*" />
+        <Bug pattern="MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR"/>
+    </Match>
+
+    <!-- Allowing Eclipse plug-in static fields using public static methods -->
+	<Match>
+		<Class name="~org\.openjdk\.jmc\..*Plugin" />
+		<Or>
+		   <Method name="getDefault" />
+		</Or>
+		<Bug pattern="MS_EXPOSE_REP" />
+	</Match>
+    <Match>
+        <Or>
+            <!-- getDefault() -->
+            <Class name="org.openjdk.jmc.ui.common.labelingrules.NameConverter"/>
+            <!-- getSecurityManager() -->
+            <Class name="org.openjdk.jmc.ui.common.security.SecurityManagerFactory"/>
+            <!-- getInstance() -->
+            <Class name="org.openjdk.jmc.rjmx.preferences.JMXRMIPreferences"/>
+            <!-- getDefault() / getPluginLogger() -->
+            <Class name="org.openjdk.jmc.browser.attach.BrowserAttachPlugin"/>
+            <!-- getInstance() -->
+            <Class name="org.openjdk.jmc.browser.attach.LocalDescriptorProvider"/>
+            <!-- getLogger() -->
+            <Class name="org.openjdk.jmc.flightrecorder.configuration.ConfigurationToolkit"/>
+            <!-- getDefault() -->
+            <Class name="org.openjdk.jmc.flightrecorder.ui.FlightRecorderUI"/>
+            <!-- createSingleton() -->
+            <Class name="org.openjdk.jmc.console.agent.manager.model.PresetRepositoryFactory"/>
+            <!-- getDefault() / getLogger() -->
+            <Class name="org.openjdk.jmc.console.jconsole.Activator"/>
+            <!-- getDefault() -->
+            <Class name="org.openjdk.jmc.console.pde.Activator"/>
+            <!-- getDefault() -->
+            <Class name="org.openjdk.jmc.flightrecorder.controlpanel.ui.ControlPanel"/>
+            <!-- get*Delegate() -->
+            <Class name="org.openjdk.jmc.flightrecorder.controlpanel.ui.model.VolatileStorageDelegate"/>
+            <!-- getDefault() -->
+            <Class name="org.openjdk.jmc.flightrecorder.pde.Activator"/>
+            <!-- getUpdateSites() -->
+            <Class name="org.openjdk.jmc.rcp.application.p2.UpdateSiteURLToolkit"/>
+            <!-- getLogger() -->
+            <Class name="org.openjdk.jmc.rcp.logging.LoggingToolkit"/>
+        </Or>
+        <Bug pattern="MS_EXPOSE_REP" />
+    </Match>
+
+    <!-- Allowing catching a NullPointerException for a JBoss 4.2.3 specific workaround -->
+    <Match>
+        <Class name="org.openjdk.jmc.rjmx.internal.RJMXConnection"/>
+		<Method name="getMBeanInfos" />
+        <Bug pattern="DCN_NULLPOINTER_EXCEPTION" />
+    </Match>
+
+    <!-- Allowing catching a NullPointerException that could occur when VM dies -->
+    <Match>
+        <Class name="org.openjdk.jmc.attach.AttachToolkit"/>
+		<Method name="importFromPid" />
+        <Bug pattern="DCN_NULLPOINTER_EXCEPTION" />
+    </Match>
+
+    <!-- Allowing catching a NullPointerException that has no specific reason?? -->
```

Comment:
I'd skip the "??".

Reply:
@thegreystone added an according comment to the actual class